### ISSUE TITLE
Uninitialized constant Fog::Compute::VcloudDirector::VappTemplate in 0.2.0

### ISSUE THREAD

**Issue by tumido** [xlab-si/fog-vcloud-director#33]
Backstory:
Hi, when I rebased my core ManageIQ repo to the latest I see certain failure when indexing our STI. Digging deep I found out it's caused by update of this gem to version 0.2.0 in https://github.com/ManageIQ/manageiq-providers-vmware/commit/41b450f9ab62f64637f37e2e4cd67f4080cd244e , before this commit everything works flawlessly.

Description:
When updated to 0.2.0 my setup fails to import  `fog/vcloud_director` at all:

```
require 'fog/vcloud_director'
NameError: uninitialized constant Fog::Compute::VcloudDirector::VappTemplate
Did you mean?  Fog::Compute::VcloudDirector::VappTemplates
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director/models/compute/vapp_templates.rb:10:in `<class:VappTemplates>'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director/models/compute/vapp_templates.rb:6:in `<class:VcloudDirector>'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director/models/compute/vapp_templates.rb:5:in `<module:Compute>'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director/models/compute/vapp_templates.rb:4:in `<module:Fog>'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director/models/compute/vapp_templates.rb:3:in `<top (required)>'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director.rb:13:in `block in <top (required)>'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director.rb:13:in `each'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/gems/2.4.0/gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director.rb:13:in `<top (required)>'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `require'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:133:in `rescue in require'
        from /home/______/.rbenv/versions/2.4.4/lib/ruby/2.4.0/rubygems/core_ext/kernel_require.rb:39:in `require'
        from (irb):1
        from /home/______/.rbenv/versions/2.4.4/bin/irb:11:in `<main>'
```

It fails using both `rails c` console and `irb`. Any idea what can be causing it?

Also note that the `vapp_template.rb` is present in `.../gems/fog-vcloud-director-0.2.0/lib/fog/vcloud_director/models/compute/`, but it's obviously not resolved properly.

Environment:
- ruby 2.4.4
- rbenv

**Comment by miha-plesko** [comment]
@tumido thanks for report, I suspect this could be related to importing [order](https://github.com/xlab-si/fog-vcloud-director/blob/master/lib/fog/vcloud_director.rb#L12-L15). Let me fix this first thing in the morning, sorry for the trouble.

**Comment by miha-plesko** [comment]
@tumido issue should be resolved in `v0.2.1` which is already available in rubygems. Therefore running

```bash
$ bundle update
```

in MIQ core repo should resolve your problem.
